src/components/outline-nav/components/outline-list-items/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { OutlineLink, OutlineLinkWithNesting } from '..'
+import type { OutlineLinkItem } from 'components/outline-nav/types'
+
+/**
+ * Render an array of OutlineNav list items.
+ *
+ * Each item can be either a plain link item, or
+ * a link item with nested links.
+ *
+ * All items are wrapped in an <li /> element.
+ */
+function OutlineListItems({ items }: { items: OutlineLinkItem[] }) {
+	return (
+		<>
+			{items.map((item: OutlineLinkItem) => {
+				if ('items' in item) {
+					return (
+						<li key={item.url}>
+							<OutlineLinkWithNesting {...item} />
+						</li>
+					)
+				} else {
+					return (
+						<li key={item.url}>
+							<OutlineLink {...item} />
+						</li>
+					)
+				}
+			})}
```

Comment:
🎨 Totally optional! Might be a place to shorten the code a little if you want.

```tsx
			{items.map((item: OutlineLinkItem) => {
				return (
					<li key={item.url}>
						{'items' in item ? (
							<OutlineLinkWithNesting {...item} />
						) : (
							<OutlineLink {...item} />
						)}
					</li>
				)
			})}
```

Reply:
Oooo yeah great call! Added this in, ty for the suggestion 💖